pysmarthashtag/const.py

```
@@ -14,11 +15,77 @@
+INTL_USER_INFO_URL = f"{INTL_API_BASE}/uc/api/user/toc/base/info"
+INTL_CA_KEY = "204587190"  # X-Ca-Key header value for INTL API Gateway
+INTL_APP_ID = "SMARTAPP-ISRAEL"  # x-app-id for vehicle API
+INTL_OPERATOR_CODE = "SMART-ISRAEL"  # x-operator-code for vehicle API
+# Note: INTL signing secret is stored in api/utils.py
+
 OTA_SERVER_URL = "https://ota.srv.smart.com/"
 
 HTTPX_TIMEOUT = 30.0
 
 
+class SmartRegion(str, Enum):
```

Ruff (0.15.2)

[error] Undefined name `Enum`

(F821)